TestHelpers.Tests/SkipOnUnixAttribute.cs

```diff
@@ -0,0 +1,36 @@
+using System;
+using System.Linq;
+using NUnit.Framework;
+using NUnit.Framework.Interfaces;
+
+namespace System.IO.Abstractions.TestingHelpers.Tests
+{
+    internal static class SkipReason
+    {
+        public const string NoDrivesOnUnix = "Unix does not have the concept of Drives";
```

Comment:
I think there are a couple tests that are skipped due to Illegal characters

Reply:
Yes, added a new reason for that